client/src/components/VideoCall.jsx

```diff
@@ -7,7 +7,9 @@ export function VideoCall ({playerName, roomName}) {
     function handleJitsiIFrameRef (iframeRef) {
         iframeRef.style.border = '10px solid cadetblue';
         iframeRef.style.background = 'cadetblue';
-        iframeRef.style.height = '400px';
+        iframeRef.style.height = '550px';
```

Comment:
This is fine for now, and will get us through the week. Down the road, it would be good to see if we can make the iframe resize itself automatically to fit the window. That way participants can scale the video window by resizing their browser.